_projects/meta_review.md

```diff
@@ -0,0 +1,54 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch accepted to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and javascript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The goal of project meta-review is to build a meta-review track system similar to meta-moderation system that can track emoji responses to review comments. These responses would be collected, processed and displayed on [gh-board][https://github.com/coala/gh-board], which is a nice serverless kanban board.
```

Comment:
maybe `s|would be collected|are to be collected|` because the rest of your sentences are in present tense